backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepositoryFactory_deriveSyncToRemoteStatusTestSuite.java

```diff
@@ -29,8 +30,24 @@ private SyncToRemoteStatus invokeDeriveSyncToRemoteStatus(IGitCoreLocalBranch co
   }
 
   @Test
+  @SneakyThrows
+  public void deriveSyncToRemoteStatus_NoRemotes() {
+    // given
+    PowerMockito.doReturn(List.empty()).when(gitCoreRepository).deriveAllRemoteNames();
+
+    // when
+    SyncToRemoteStatus syncToRemoteStatus = invokeDeriveSyncToRemoteStatus(coreLocalBranch);
+
+    // then
+    Assert.assertEquals(SyncToRemoteStatus.Relation.NoRemotes, syncToRemoteStatus.getRelation());
+  }
+
+  @Test
+  @SneakyThrows
   public void deriveSyncToRemoteStatus_Untracked() {
     // given
+    PowerMockito.doReturn(List.of("origin")).when(gitCoreRepository).deriveAllRemoteNames();
```

Comment:
what about introducing a constant for `List.of("origin")`?

Reply:
Extracted `"origin"` to a constant